superset-frontend/webpack.proxy-config.js

```diff
@@ -30,7 +30,7 @@ const parsedEnvArg = () => {
 };
 
 const { supersetPort = 8088, superset: supersetUrl = null } = parsedEnvArg();
-const backend = (supersetUrl || `http://localhost:${supersetPort}`).replace(
```

Comment:
Is this fixing anything? Someone might configure localhost to be at another IP and might be (marginally) safer.

Reply:
+1

Reply:
https://github.com/apache/superset/pull/28440#issuecomment-2107935034

Reply:
Hah! The answer was _right there_! 🤦 What a goofy change for them to make. 

I wonder if forcing IPV4 like this is risky. Some might change that IP, or some might _want_ IPV6 support. 

Maybe it's worth a big find/replace to use "::" rather than "0.0.0.0" everywhere, so that it'll (supposedly) support both?

Reply:
I dug deeper into the rabbit hole and Node 20 should be able to fix the IPv6-over-IPv4 issues automatically, thanks to this [PR](https://github.com/nodejs/node/pull/44731). 

I tried reverting the `localhost` -> `127.0.0.1` change, run `npm ci && npm run dev-server` and voila, `localhost:9000` is proxied properly!

So yeah, this change can wait until Node20 is chosen for `superset-frontend`'s .nvmrc :D

Reply:
Good to know! @michael-s-molina and I were talking about making this bump as part of the 5.0 release rather than rocking the boat beforehand. I'll try to open a PR with Node 20 sooner rather than later, and just make sure we don't merge it too soon :)